src/database/upgrade/25/openfire_db2.sql

```diff
@@ -0,0 +1 @@
+UPDATE ofVersion SET version = 25 WHERE name = 'openfire';
```

Comment:
You're not creating the new index in any of the upgrade scripts, but you did add them to the initial installation scripts - is that intended?